client/layout/index.jsx

```diff
@@ -504,11 +504,21 @@ export default withCurrentRoute(
 		const userAllowedToHelpCenter =
 			config.isEnabled( 'calypso/help-center' ) && ! getIsOnboardingAffiliateFlow( state );
 
-		const colorScheme = getColorScheme( {
-			state,
-			sectionName,
-			isGlobalSidebarVisible,
-		} );
+		let colorScheme = null;
+
+		// Try to get the color scheme from `color_scheme` query param if we're in the WooCommerce Passwordless JPC flow.
+		if ( isWooPasswordlessJPC ) {
```

Comment:
How about using the ternary operator here to simplify the logic?

```js
let colorScheme =
    isWooPasswordlessJPC
        ? getColorSchemeFromCurrentQuery(currentQuery)
        : getColorScheme({
              state,
              isGlobalSidebarVisible,
              sectionName,
          });
```

Reply:
I was tempted to use ternary at first.

When `color_scheme` query param  is missing for some reason, I think we should call `getColorScheme` to get the user's preference instead.

Reply:
@adrianduffell Good catch! Let's use the default color scheme in that case.

Reply:
@chihsuan @adrianduffell Fixed it in [Return 'default' color scheme](https://github.com/Automattic/wp-calypso/pull/96557/commits/ef3813fdd41011a18c8cfe2e7952ff4c46cf441a)